bedrock/firefox/templates/firefox/includes/download-button.html

```diff
@@ -50,6 +50,46 @@
       <p>{{ ftl('download-button-your-system-may') }}</p>
       {{ alt_buttons(builds) }}
     </div>
+
+    {% if channel == 'beta' %}
+      {% set channel_name = 'Firefox Beta' %}
+    {% elif channel == 'alpha' %}
+      {% set channel_name = 'Firefox Developer Edition' %}
+    {% elif channel == 'nightly' %}
+      {% set channel_name = 'Firefox Nightly' %}
+    {% else %}
+      {% set channel_name = 'Firefox' %}
+    {% endif %}
+
+    <div class="fx-unsupported-message win" data-nosnippet="true">
+      <p><strong>{{ ftl('download-button-unsupported-platform', channel_name=channel_name, help_url='https://support.mozilla.org/kb/firefox-users-macos-1012-1013-1014-moving-to-extended-support', os_version='Windows 8.1') }}</strong></p>
```

Comment:
Why is this a mac link but os_version references Windows 8.1?

Reply:
This message is for Windows users

Reply:
But the help_url points to a MacOS-specific SUMO page?

Reply:
Ah, the href is the wrong way around - nice catch!